Change currency SLL/694 to SLE/925 (#152)
clearhaus/gateway-api-docs: website/content/gateway/api_reference/changes/changes.md

Sorted by descending timestamp

#### Change SLL currency to SLE

> The currency SLL will be changed to SLE after 2022-10-07.

Author: ```suggestion
The currency SLL has been changed to SLE on 2022-10-07.
```

Or something like that :slightly_smiling_face: 

:man_shrugging:
